docs/deploy-on-k8s.md

```diff
@@ -0,0 +1,115 @@
+
+## Using the application
+
+So now you have the application installed in Kubernetes, how can you use it? The standard practise for accessing a service inside a Kubernetes cluster would be to create a Service with type LoadBalancer. On AWS, for example, this would cause AWS to automatically provision an ELB or ALB for you and connect it to the Kubernetes service, allowing external access. As we are only providing a demonstration system and don't wish to make things available over the public internet, we are not using LoadBalancer in our deployment. Instead we provide some scripts which use Kubernetes port-forwarding so that you can reach the Services in Kubernetes from your local machine.
+
+To see the application's Web UI you can run the script:
+
+```sh
+./deployments/port-forward-web.sh
+```
+
+That command will continue to run until you kill it (with Ctrl+C) for example. While the command is running it will forward any traffic from your localmachine on port 3000 to the application's Web UI in the cluster. You should now be able to see the application at: http://localhost:3000/
+
+We also provide two other port forwarding scripts. The first is to give you access to the web UI for the Temporal cluster running in your cluster:
+
+```sh
+./deployments/port-forward-temporal-ui.sh
+```
+
+While this command is running you will be able to see the Temporal Web UI at: http://localhost:8080. Try placing an order with the application's Web UI and then looking at the Temporal Web UI to see which workflows were created.
+
+You may notice on the Temporal Web UI that the inputs and results for the Workflows and Activities are encrypted, due to the application's use of an encrypting data converter. In order to decrypt those on the Web UI so that you can see what the inptus and results were, you can use a codec server which we deployed alongside the app.
+
+The last of our port-forwarding commands is to allow you access to the codec server:
+
+```sh
+./deployments/port-forward-codec-server.sh
+```
+
+Leave this command running and browse to the Temporal Web UI at http://localhost:8080. Now you can configure the Temporal Web UI to use the application's codec server to decrypt inputs and results by clicking on the sunglasses icon top right of the page. In the pop-up that displays, enter "http://localhost:8089" into the "Codec Server browser endpoint" field, and hit "Apply". You should now be able to navigate to one of the workflows and see the inputs and results in plaintext. Remember: the codec server will only work if the port forwarding command is running, otherwise your browser will not be able to access the codec server. In this case you would see encrypted content again, and the sunglasses icon on the Temporal UI will turn red to indicate a problem talking to the codec server. Just start the port forward command again, and the Temporal UI will then be able to reconnect next time you navigate to a workflow page (or refresh the one you are on).
```

Comment:
```suggestion
Leave this command running and browse to the Temporal Web UI at http://localhost:8080. Now you can configure the Temporal Web UI to use the application's codec server to decrypt inputs and results by clicking on the sunglasses icon top right of the page. In the pop-up that displays, enter "http://localhost:8089" into the **Codec Server browser endpoint** field, and click the **Apply** button. You should now be able to navigate to one of the workflows and see the inputs and results in plaintext. Remember: the codec server will only work if the port forwarding command is running, otherwise your browser will not be able to access the codec server. In this case you would see encrypted content again, and the sunglasses icon on the Temporal UI will turn red to indicate a problem talking to the codec server. Just start the port forward command again, and the Temporal UI will then be able to reconnect next time you navigate to a workflow page (or refresh the one you are on).
```